src/pcp/atop/atop.c

```diff
@@ -588,6 +588,7 @@ engine(void)
 				cleanstop(1);
 			goto reset;
                }
+		set_system_btime(cursstat->cpu.btime);
```

Comment:
@shogo-matsumoto hmm, could this be done in various.c::setup_globals() rather than on every fetch iteration?

Reply:
@natoscott Thank you for your feedback! It seems good idea but I'm concerned about if it works for an archive having multiple boot time which is generated by pmlogextract.

Reply:
@shogo-matsumoto yes, you're right - needs to be checked on each sample then.  I still recommend use the 'boottime' global variable name, and updating it directly in photosyst(), rather than using a function called at the top level in atop.c::engine().

Reply:
@natoscott Thank you for the reply.
> I still recommend use the 'boottime' global variable name
Yes, I agree.